platform/ios/src/MGLMapView.h

```diff
@@ -105,6 +105,15 @@ typedef NS_ENUM(NSUInteger, MGLUserTrackingMode) {
     MGLUserTrackingModeFollowWithCourse,
 };
 
+typedef NS_ENUM(NSUInteger, MGLUserScrollingMode) {
+    /** The map allows the user to only scroll horizontally. */
+    MGLUserScrollingModeHorizontal              = 0,
+    /** The map allows the user to only scroll vertically. */
+    MGLScrollingModeVertical,
+    /** The map allows the user to scroll both horizontally and vertically. */
+    MGLScrollingModeUnlimited
+};
+
```

Comment:
Should this be `MGLUserScrollingMode` or `MGLScrollingMode`?

Reply:
Given that this only affects the pan gesture, perhaps we include `Pan` in there somewhere?

Longer term, this property feels like it would better live with a custom pan gesture that we expose.